broadinstitute/pearl
# [JN-504] unlocking survey editing for regular users

Allows regular staff users to save edited surveys.  This does not allow regular users to publish surveys to non-sandbox environments.

TO TEST:
1. sign into localhost:3000 using development mode as `staff@ourhealth.org`
2. go to `https://localhost:3000/ourhealth/studies/ourheart/env/sandbox`
3. pick a survey to edit, edit it, and save
4. confirm it saves and the environment updates
